pylint/pyreverse/inspector.py

```diff
@@ -229,8 +230,17 @@ def handle_assignattr_type(node, parent):
 
         handle instance_attrs_type
         """
+        ann = None
```

Comment:
```suggestion
        ann = node.infer()
```
I think doing that would simplify that following code.